Dockerfile.centos.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/Dockerfile b/Dockerfile
+index 23b9fce7..a480fe28 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -1,4 +1,4 @@
+-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
++FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```

Comment:
The minimum golang version here should be `1.14` for this patch file. We use couple of 1.14-only features in our code base AFAIK.

Reply:
+1. Also I am keeping RHEL as the default, the HACKING.md explains how to use the centos patch.